catalog/dags/common/urls.py

```diff
@@ -33,6 +37,9 @@ def validate_url_string(url_string, strip_slash: bool = True):
     strip_slash: Flag (bool) decides whether to strip slashes in URL or not,
     Default value is `True`
     """
+    if url_string and " " in url_string:
+        raise SpaceInUrlError("space is present in URL")
```

Comment:
Can we also log the `url_string` in the error for debugging purposes?

Reply:
Yes, I will add `url_string` in the error message.